server.js

```diff
@@ -0,0 +1,24 @@
+require('./src/config');
+const express = require('express');
+const argv = require('minimist');
+const path = require('path');
+const healthcheck = require('express-healthcheck');
+
+const args = argv(process.argv.slice(2));
+const HTTP_PORT = process.env.PORT || args.port || 3000;
+const server = express();
```

Comment:
Let's follow best practices and break the definition of express app away from the web server, see https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/separateexpress.md.  This will make testing a lot easier.